### PR TITLE
Removed Google +

### LINKED ISSUE #1065: Remove Google +
## Expected Behavior
Game should not have links to Google Plus.

## Current Behavior
The game has links to Google + which is not required as it is extinct.

## Possible Solution
Remove Google +

## Steps to Reproduce (for bugs)

1.
2.
3.
4.

## Your Environment

* Browser Name and version:
* Link to your project:
* Machine Configuration: Ram, Graphics Memory

### PR DESCRIPTION
**Pull request by NishithP2004**
- [x] Read and understood [see CONTRIBUTING.md](https://github.com/fossasia/labyrinth/blob/master/CONTRIBUTING.md)
- [x] Included a preview link and screenshot showing after and before the changes.
- [x] Included a description of change below.
- [x] Tried to [squash the commits][squash].
- [ ] Added Screenshot / Path to tile to Indicate Location of Change in Game

### Changes done in this Pull Request
Removed Google Plus
- View my work at https://pr-1063-fossasia-labyrinth.surge.sh

- Fixes #1065 

### Screenshots,Path to Tile

[squash]: https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git

### CODE REVIEW

**Comment by YashKumarVerma** [comment]
It would be great if you could open an issue for the same / link an existing one to this PR.

**Comment by NishithP2004** [comment]
> It would be great if you could open an issue for the same / link an existing one to this PR.

Done

**Comment by YashKumarVerma** [comment]
> > It would be great if you could open an issue for the same / link an existing one to this PR.
> 
> Done

the commits need to be squashed

**Comment by NishithP2004** [comment]
> > > It would be great if you could open an issue for the same / link an existing one to this PR.
> > 
> > 
> > Done
> 
> the commits need to be squashed

Done,Please reviww.

File: img_config.yml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 - directory: ./tiles/rooms/floor/*.*
-  width: 429.544
-  height: 256.314
+  width: 429.544px
+  height: 256.314px
```
Comment (YashKumarVerma) [inline img_config.yml]: are you sure that adding characters to the gem config file won't err the script?

Comment (NishithP2004) [reply img_config.yml]: Sorry,i didn't notice that.

**Comment by YashKumarVerma** [comment]
The commits need to be squashed.

**Comment by NishithP2004** [comment]
> The commits need to be squashed.

Please review, i have merged the commits.

**Pull request merged**